packages/flutter/lib/src/rendering/box.dart

```diff
@@ -2,7 +2,6 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
```

Comment:
(applies to many files) when removing this line we need to also remove the blank line we'd added so there's only one blank line left after the license

Reply:
Will change